pkg/route/api/validation/validation.go

```diff
@@ -300,9 +301,14 @@ func validateCertificatePEM(certPEM string, options *x509.VerifyOptions) (*x509.
 	}
 
 	if options != nil {
+		// Ensure we don't report errors for expired certs or if
+		// the validity is in the future.
+		options.CurrentTime = cert.NotAfter.AddDate(0, 0, -1)
```

Comment:
looks like the intermediate certs are failing... if we have a cert valid from A-B signed by another cert valid from C-D, and A-B/C-D are non-overlapping, that chain will not be valid at any single instant.